packages/vite/src/node/plugins/asset.ts

```diff
@@ -36,6 +42,76 @@ export function registerCustomMime(): void {
   mrmime.mimes['eot'] = 'application/vnd.ms-fontobject'
 }
 
+export function renderAssetUrlInJS(
+  ctx: PluginContext,
+  config: ResolvedConfig,
+  chunk: RenderedChunk,
+  opts: NormalizedOutputOptions,
+  code: string
+): MagicString | undefined {
+  let match: RegExpExecArray | null
+  let s: MagicString | undefined
+
+  // Urls added with JS using e.g.
+  // imgElement.src = "__VITE_ASSET__5aa0ddc0__" are using quotes
+
+  // Urls added in CSS that is imported in JS end up like
+  // var inlined = ".inlined{color:green;background:url(__VITE_ASSET__5aa0ddc0__)}\n";
+
+  // In both cases, the wrapping should already be fine
+
+  while ((match = assetUrlRE.exec(code))) {
+    s ||= s = new MagicString(code)
```

Comment:
```suggestion
    s ||= new MagicString(code)
```
Though it still does the same thing, I think this is less confusing.

Reply:
Oh, yes, @poyoho let's swap to `??=` also for both so we don't need to do it in a future PR

Reply:
`||=` to `??=` ?

Reply:
`??=` to `||=` ?